bats/tests/helpers/utils.bash

```diff
@@ -19,6 +20,10 @@ bool() {
     fi
 }
 
+assert_nothing() {
```

Comment:
It would be good to have a comment about what this does. (Specifically, that it doesn't assert anything, and not that it asserts _something_ is nothing).

… I hope you can parse that.

Reply:
Good idea, will add a comment